internal/commands/platform/export_internal.go

```diff
@@ -212,6 +212,13 @@ func initPingFederateServices(ctx context.Context, pingcliVersion string) (err e
 		return fmt.Errorf("failed to initialize PingFederate services. unrecognized authentication type '%s'", authType)
 	}
 
+	// Test PF API client with create Context Auth
+	_, response, err := pingfederateApiClient.VersionAPI.GetVersion(pingfederateContext).Execute()
+	ok, err := common.HandleClientResponse(response, err, "GetVersion", "pingfederate_client_init")
+	if err != nil || !ok {
+		return fmt.Errorf("failed to initialize PingFederate Go Client. Check authentication type and credentials")
```

Comment:
Does `err` have anything useful that would be helpful to include in this message?

Reply:
I don't want the user getting a "red herring" on the /version endpoint issue, so I overwrote the error message here to avoid that.